downstream/assemblies/hub/assembly-deploying-high-availability-hub.adoc

```diff
@@ -0,0 +1,9 @@
+[id="assembly-deploying-high-availability-hub"]
+
+= Requirements for a high availability {HubName}
+
+Ensure that the following requirements are met as you configure the {PlatformNameShort} installer. Before you run the {PlatformNameShort} installer, you will need to make several changes to the inventory file found in the {PlatformNameShort} installer, as well as set up other components necessary to deploy in HA. For more information about installing automation hub using the {PlatformNameShort} installer, please see https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1#[the AAP installation guides].
```

Comment:
I would change the URL text to the full platform name and avoid using the acronym, since I do not think we're allowed to do so.